providers/redfish/firmware_test.go

```diff
@@ -112,17 +112,17 @@ func TestFirmwareInstall(t *testing.T) {
 			"invalid applyAt parameter",
 			"applyAt parameter invalid",
 		},
-		{
-			common.SlugBIOS,
-			constants.FirmwareApplyOnReset,
-			false,
-			true,
-			fh,
-			"467696020275",
-			bmclibErrs.ErrFirmwareInstall,
-			"task for BIOS firmware install present",
-			"task ID exists",
-		},
+		//{
```

Comment:
This test is most likely failing because of this section being commented out - https://github.com/bmc-toolbox/bmclib/blob/f096534cec29f6fc4c415a698bd883bdcf5b4894/providers/redfish/firmware.go#L65